packages/headless/esbuild.mjs

```diff
@@ -87,15 +87,23 @@ const browserEsm = Object.entries(useCaseEntries).map((entry) => {
   const [useCase, entryPoint] = entry;
   const outDir = getUseCaseDir('dist/browser', useCase);
   const outfile = `${outDir}/headless.esm.js`;
+  const config = {
```

Comment:
nitpicky: I would just modify the `config` used based on `devMode` value instead of calling `buildBrowserConfig` differently with override on this config.

Reply:
done